design/AdapterInfoRegistry.md

```diff
@@ -0,0 +1,46 @@
+# WebGPU Adapter Info Registry
+
+This living document lists `"vendor"` and `"architecture"` adapter information values in the WebGPU API returned by the `requestAdapterInfo()` method on a `GPUAdapter` instance (See more at [AdapterIdentifiers.md](https://github.com/gpuweb/gpuweb/blob/main/design/AdapterIdentifiers.md)).
+
+## JS example
+
+```js
+const gpuAdapter = await navigator.gpu.requestAdapter();
+const adapterInfo = await gpuAdapter.requestAdapterInfo();
+console.log(adapterInfo);
+
+// Output:
+{
+    vendor: "nvidia",
+    architecture: "turing",
+    device: "",
+    description: ""
+}
+```
+
+## Values for `"vendor"` and `"architecture"` in Chromium
```

Comment:
If we can get agreement to have a registry, I'd like to ideally have a more prescriptive registry (rather than describing behavior in different browsers). Essentially, saying, if a browser is going to report an architecture, and the architecture it wants to report exists in the registry, it **should** use the spelling in the registry.